packaging/src/docker/docker-compose.yml

```diff
@@ -32,6 +32,9 @@ services:
         - '9083:9083'
     volumes:
         - warehouse:/opt/hive/data/warehouse
+        - type: bind
+          source: ${MAVEN_LOCAL_REPOSITORY}/org/postgresql/postgresql/42.5.1/postgresql-42.5.1.jar
```

Comment:
does it assumes the maven local repo will be having postgresql-42.5.1.jar always?

Reply:
Currently we use postgres 42.5.1 in our project, it's easy for user to change the version or download the jar as they wish.

Reply:
For me it is printing
```
WARN[0000] The "MAVEN_LOCAL_REPOSITORY" variable is not set. Defaulting to a blank string. 

```

So, I need to set ``MAVEN_LOCAL_REPOSITORY`` explicitly first to use it?

And then it kind of got hung creating metastore
```
WARN[0000] The "MAVEN_LOCAL_REPOSITORY" variable is not set. Defaulting to a blank string. 
[+] Building 0.0s (0/0)                                                                                                                                                                       
[+] Running 1/2
 ✔ Container postgres   Created                                                                                                                                                          0.0s 
 ⠧ Container metastore  Creating                                                                                                                                                           103.5s  

```

It is 100+ seconds and still going on as I write this

Reply:
Should we download the postgres jar automatically while spinning up the cluster?

Reply:
do you run with this command?
```shell
MAVEN_LOCAL_REPOSITORY=`mvn help:evaluate -Dexpression=settings.localRepository -q -DforceStdout` docker compose up -d ?

Reply:
```
ayushsaxena@ayushsaxena docker % export HIVE_VERSION=4.0.0-beta-1                                                                                       
ayushsaxena@ayushsaxena docker % MAVEN_LOCAL_REPOSITORY=`mvn help:evaluate -Dexpression=settings.localRepository -q -DforceStdout` docker compose up -d 
[+] Running 15/15
 ✔ hiveserver2 Pulled                                                                                                                                                                   61.0s 
 ✔ metastore 13 layers [⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿]      0B/0B      Pulled                                                                                                                           61.0s 
   ✔ a9fe95647e78 Already exists                                                                                                                                                         0.0s 
   ✔ 4015b6e8cc8d Already exists                                                                                                                                                         0.0s 
   ✔ 0e86b181efa0 Already exists                                                                                                                                                         0.0s 
   ✔ 94abd992e68d Already exists                                                                                                                                                         0.0s 
   ✔ e7a78026257f Pull complete                                                                                                                                                         54.0s 
   ✔ 3874402c80d6 Pull complete                                                                                                                                                         56.1s 
   ✔ a8c0cb8787a3 Pull complete                                                                                                                                                         56.2s 
   ✔ 7e494428637a Pull complete                                                                                                                                                         56.2s 
   ✔ 64e47b23d332 Pull complete                                                                                                                                                         56.2s 
   ✔ f9c33c65d8ef Pull complete                                                                                                                                                         56.2s 
   ✔ 885d438fb370 Pull complete                                                                                                                                                         56.2s 
   ✔ 340aa3eab0dc Pull complete                                                                                                                                                         56.2s 
   ✔ 4f4fb700ef54 Pull complete                                                                                                                                                         56.3s 
[+] Building 0.0s (0/0)                                                                                                                                                                       
[+] Running 0/1
 ⠏ Container metastore  Creating                                                                                                                                                        136.7s 


```

It got hung again, I think because in my maven local repo there is no postgres jar

Reply:
Hi @ayushtkn, I have updated the command, which tries to download the postgres jar into maven local repo, cloud you run this command please?